js/page-specific/swiftype-search-results.js

```diff
@@ -17,52 +17,76 @@ baltimoreCounty.pageSpecific.swiftypeSearchResults = (function swiftypeSearchRes
 		console.error(err);
 	}
 
-	function searchResultRequestSuccessHandler(response) {
-		var info = response.info.page;
-		var hits = response.records.page;
-		var maxPages = 10;
-		var tooManyResults = info.num_pages > maxPages;
-		var lastPage = tooManyResults ? maxPages : info.num_pages;
-		var searchResults = [];
-		var pageLinks = [];
-		var lastPageNumber = info.current_page * info.per_page < info.total_result_count ?
-			info.current_page * info.per_page :
-			info.total_result_count;
-		var firstPageNumber = ((info.current_page - 1) * info.per_page) + 1
-
-		info.base_url = window.location.pathname + '?q=' + info.query + '&page=';
-
-		info.index = {
-			first: firstPageNumber,
-			last: lastPageNumber
-		};
+	function buildSearchResults(hits) {
+		var results = [];
 
 		$.each(hits, function eachHit(index, hit) {
```

Comment:
Not a killer but could we refactor this into it's own function with a meaningful name?

Reply:
Done!